server/src/gameplay/event.rs

```diff
@@ -65,9 +70,21 @@ pub enum Event {
     },
 
     /* -- STATUS PHASE EVENTS -- */
+    ScorePublicObjective {
+        player: PlayerId,
+        objective: Objective,
+    },
+
+    ScoreSecretObjective {
+        player: PlayerId,
+        objective: SecretObjective,
+    },
+
     // TODO: Score objectives & Reveal objectives
     CompleteStatusPhase,
     // TODO: Agenda phase
+
+    /* -- ANY PHASE EVENTS -- */
```

Comment:
What's this about?

Reply:
uh, i was gonna add stuff here but then I didn't :p